docs/source/models/risk_exposures/child_growth_failure/wasting_state_exposure/index.rst

```diff
@@ -112,82 +121,559 @@ Wasting, a sub-compoonent indicator of child growth failure (CGF), is based on a
 | SAM   |  < -3 Z score                        |
 +-------+--------------------------------------+
 
-.. _waste_exp1.1.2:
+
+Exposure estimation
+-------------------
+
+In modeling CGF, all data types go into ST-GPR modeling. GBD has STGPR models 
+for moderate, severe, and mean stunting, wasting, and underweight. The output 
```

Comment:
"moderate, severe, and mean" seems like it might be wrong, just flagging in case it is "mild, moderate, and severe" or something more similar than to categories and a continuous.